packages/app-desktop/app.ts

```diff
@@ -566,22 +566,6 @@ class Application extends BaseApplication {
 			value: Setting.value('flagOpenDevTools'),
 		});
 
-		// Note: Auto-update is a misnomer in the code.
-		// The code below only checks, if a new version is available.
-		// We only allow Windows and macOS users to automatically check for updates
-		if (shim.isWindows() || shim.isMac()) {
-			const runAutoUpdateCheck = () => {
-				if (Setting.value('autoUpdateEnabled')) {
-					void checkForUpdates(true, bridge().window(), { includePreReleases: Setting.value('autoUpdate.includePreReleases') });
-				}
-			};
-
-			// Initial check on startup
-			shim.setTimeout(() => { runAutoUpdateCheck(); }, 5000);
-			// Then every x hours
-			shim.setInterval(() => { runAutoUpdateCheck(); }, 12 * 60 * 60 * 1000);
-		}
-
```

Comment:
Here we have still the same problem - we actually want to keep this code, and it should be enabled by default. The new AutoUpdaterService should be disabled by default.

What I meant with the feature flag is that you should create a new setting, for example `featureFlag.autoUpdaterServiceEnabled`, which defaults to `false`. And you use this to enabled/disable the old code and disable/enable the new code.

Have a look in Setting.ts - there are some examples of commented out feature flags (they start with `featureFlag.`). Create a similar setting value and use it here.

We need this for safety - because if there's any issue with your change, a lot of people won't be notified any more when a new version is available. So by default everybody use the old method, and the user can explicitly enable the new method (like I will do for example)